### ISSUE TITLE
mlnet command will not run - wrong sdk in tutorial

### ISSUE THREAD

**Issue by liranbd1** [dotnet/machinelearning-modelbuilder#1988]
**System Information (please complete the following information):**
 - Model Builder or CLI Version: 
 - Visual Studio Version (if applicable): Using Terminal but have Visual Studio 2019 for mac

**Describe the bug**
- On which page of the tutorial did you run into an issue (URL): https://dotnet.microsoft.com/en-us/learn/ml-dotnet/get-started-tutorial/install
- Clear description of the problem: When trying to validate that mlnet command is working got a message for .NetCore 3.1 (arm64).
Got two links to download the required framework
1. https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=3.1.0&arch=arm64&rid=osx.12-arm64
2. https://download.visualstudio.microsoft.com/download/pr/2dc016eb-cb93-4bdf-98c0-ee8c84ad26c0/d1a7eb02814ac9e164cb0e23c24d5973/dotnet-sdk-3.1.416-osx-x64.pkg

In the second link, it is downloading x64 while the tutorial is for macOS which also have M1 (arm64).
If we go into the first link, it also sends us to a page to download the .NetCore, but when you choose MacOS again we are passed to x64 page not arm64.
When looking for NetCore 3.1 there is no arm64 download in the following page https://dotnet.microsoft.com/en-us/download/dotnet/3.1 

**To Reproduce**
1. Go to the Tutorial page for ML.NET https://dotnet.microsoft.com/en-us/learn/ml-dotnet/get-started-tutorial/install
2. Follow the steps in the tutorial
3. When verifying that mlnet command is working you will get the following error 
![Screen Shot 2022-01-22 at 14 49 14](https://user-images.githubusercontent.com/9438255/150639162-6c482599-8eb0-4594-affd-1498a552c2e3.png)

**Expected behavior**

**Screenshots**
No download for arm64 
![Screen Shot 2022-01-22 at 14 51 12](https://user-images.githubusercontent.com/9438255/150639227-00a9a3d3-0168-493e-bff6-e1919615c71b.png)

**Comment by beccamc** [comment]
@liranbd1 Thanks for reporting!

**Comment by LittleLittleCloud** [comment]
Is your mac using m1 chip? If so the mlnet won't work as the document is not up-to-date with m1 chip and mlnet is still targeted to netcoreapp 3.1.

We will release a mlnet that can work with m1 soon. Meanwhile as a work around, you can try the following package

`dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.OSX --version 16.9.3`

**Comment by liranbd1** [comment]
Yes my mac is with m1 chip, it not only that the document is not up-to-date with m1 chip its also the dotnet \ mlnet cli tool is not updated.
If we call the mlnet command it will output a message letting the user know that it can't find the correct SDK and share a link which can be clearly seen should send us to an arm64 but as you guys just said it is not yet out.

Thanks, I will check this work around, any estimation on when mlnet will be updated to work with m1?

**Comment by LittleLittleCloud** [comment]
@liranbd1 
It will be soon, maybe @luisquintanilla knows more here

**Comment by luisquintanilla** [comment]
Thanks for reporting this issue @liranbd1. The tutorials have been recently updated so this issue should be resolved. Closing this issue for now. If you have additional questions or problems, feel free to open a new issue.

**Comment by Keshish** [comment]
@luisquintanilla 
I am using M1 Pro chips and i have the same issue

**Comment by luisquintanilla** [comment]
@Keshish these instructions should help unblock you.

![image](https://user-images.githubusercontent.com/46974588/214082495-743c4a32-966a-419a-886f-dac19bedb962.png)
